packages/theme/src/theme/_forms.scss

```diff
@@ -130,6 +147,10 @@ form {
 		&[multiple] {
 			background: none;
 		}
+
+		option:default {
```

Comment:
Do you have any reference?

Reply:
sample reference included in the theme http://1094.talend.surge.sh/theme/#forms